spec/features/suggested_salary_searches_spec.rb

```diff
@@ -0,0 +1,287 @@
+          search_for_salaried_courses_with_query_params(default_query_for_location_search(radius: 5))
+
+          expect(results_page.suggested_search_heading.text).to eq("Suggested searches")
+          expect(results_page.suggested_search_description.text).to eq("You can find:")
+          expect(results_page.suggested_search_links.first.text).to eq("4 courses including non salaried within 5 miles")
+          expect(results_page.suggested_search_links.last.text).to eq("10 courses within 10 miles")
+        end
+      end
+
+      context "with courses available that are non-salaried with a larger radius and salaried in a larger radius" do
+        before do
+          stub_suggested_without_salary_filter(radius: 5, number_of_results: 0)
+          stub_suggested_without_salary_filter(radius: 10, number_of_results: 4)
+          stub_suggested_with_salary_filter(radius: 10, number_of_results: 10)
+        end
+
+        it "displays a suggested search for the non-salaried courses with the larger radius first followed with salaried in a larger radius" do
+          search_for_salaried_courses_with_query_params(default_query_for_location_search(radius: 5))
+
+          expect(results_page.suggested_search_heading.text).to eq("Suggested searches")
+          expect(results_page.suggested_search_description.text).to eq("You can find:")
+          expect(results_page.suggested_search_links.first.text).to eq("4 courses including non salaried within 10 miles")
+          expect(results_page.suggested_search_links.last.text).to eq("10 courses within 10 miles")
+        end
+      end
+
+      context "no courses are found in the suggested searches" do
+        before do
+          stub_suggested_without_salary_filter(radius: 5, number_of_results: 0)
+          stub_suggested_without_salary_filter(radius: 10, number_of_results: 0)
+          stub_suggested_without_salary_filter(radius: 20, number_of_results: 0)
+          stub_suggested_without_salary_filter(radius: 50, number_of_results: 0)
+          stub_suggested_across_england_without_salary_filter(number_of_results: 0)
+
+          stub_suggested_with_salary_filter(radius: 5, number_of_results: 0)
+          stub_suggested_with_salary_filter(radius: 10, number_of_results: 0)
+          stub_suggested_with_salary_filter(radius: 20, number_of_results: 0)
+          stub_suggested_with_salary_filter(radius: 50, number_of_results: 0)
+          stub_suggested_across_england_with_salary_filter(number_of_results: 0)
+        end
+
+        it "doesn't show the link if there are no courses found" do
+          search_for_salaried_courses_with_query_params(default_query_for_location_search(radius: 5))
+
+          expect(results_page).not_to have_suggested_search_links
+        end
+      end
+    end
+
+    context "And the initial search was filtered to a 10 mile radius" do
```

Comment:
We shouldn't capitalise the first letter of contexts, its etc as they get concatenated in the results and they look funny. (sorry, I'm at the boundaries of adding value here 😆)